rootfs/api/serializers.py

```diff
@@ -326,27 +326,30 @@ def validate_registry(self, data):
         return data
 
     def validate_healthcheck(self, data):
-        for key, value in data.items():
-            if value is None:
+        for procType, healthcheck in data.items():
+            if healthcheck is None:
                 continue
+            else:
```

Comment:
Else is not needed here, drop that and bring everything in one level would be my recommendation